app/src/main/res/layout/submit_button_item.xml

```diff
@@ -54,6 +54,7 @@
       android:enabled="@{buttonViewModel.canSubmitAnswer}"
       android:onClick="@{(v) -> buttonViewModel.submitNavigationButtonListener.onSubmitButtonClicked()}"
       android:text="@string/state_submit_button"
+      android:textColor="@{buttonViewModel.canSubmitAnswer ? @color/color_def_white : @color/component_color_shared_submit_text_inactive_color}"
```

Comment:
Change `color_def_white ` too

Reply:
Also make sure that the change does support dark-mode and light-mode

Reply:
> Change `color_def_white ` too

Done